ivy_tests/test_ivy/test_frontends/test_torch/test_non_linear_activation_functions.py

```diff
@@ -104,3 +104,40 @@ def test_torch_softmax(
         dim=axis,
         dtype=dtypes[0],
     )
+
+
+@given(
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=tuple(
+            set(ivy_np.valid_float_dtypes).intersection(
+                set(ivy_torch.valid_float_dtypes)
+            )
+        )
+    ),
+    num_positional_args=helpers.num_positional_args(
+        fn_name="ivy.functional.frontends.torch.non_linear_activation_functions.gelu"
+        ),
```

Comment:
A small error here.
ivy_tests/test_ivy/test_frontends/test_torch/test_non_linear_activation_functions.py:119:9: E123 closing bracket does not match indentation of opening bracket's line